apps/frontend/src/modules/project-api-keys/slices/actions.ts

```diff
@@ -27,4 +27,16 @@ const create = createAsyncThunk<
 	return response;
 });
 
-export { create };
+const copy = createAsyncThunk<string, string, AsyncThunkConfig>(
```

Comment:
let's make it more clear
```suggestion
const copyToClipboard = createAsyncThunk<string, string, AsyncThunkConfig>(
```